src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="nav_bar">
+        <ul class="item_box">
+          <li class="item"><a class="link is-active" href="#">Apple</a></li>
```

Comment:
- It's not just `item` or `link`, they are related to `nav_bar` or `menu`(not `item_box`).
Fix naming for your classes. 
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/29022a4a-d2f5-4d5b-b1a2-da0ca0a64a22)